src/server/api.test.js

```diff
@@ -0,0 +1,74 @@
+import request from 'supertest';
+
+import app from '.';
+import * as chartData from '../utils/apiUtils/chartData';
+
+describe('GET /api/stocks/:symbol', () => {
+  let mockRequestMaxStockData;
+  let mockRequestQuote;
+  let mockRequestOneDayStockData;
+  let mockRequestFiveDayStockData;
+
+  beforeAll(() => {
+    const mockRequestImplementation = (_symbol, callback) => (
+      callback(null, {})
+    );
+
+    mockRequestMaxStockData = jest.spyOn(
```

Comment:
When I suggested putting these in beforeAll, that was when I thought they'd be used in multiple specs.  I think now they should be moved into `it('returns 200 for a valid stock symbol'` and scoped down to that spec only.

Also, is `spyOn` needed, since you're not asserting anything was called?  `jest.fn` would probably do the trick here.